[SPARK-28928][SS]Use Kafka delegation token protocol on sources/sinks (#25631)
apache/spark: docs/structured-streaming-kafka-integration.md

    <td><code>spark.kafka.clusters.${cluster}.security.protocol</code></td>
    <td>SASL_SSL</td>
    <td>
      Protocol used to communicate with brokers. For further details please see Kafka documentation. Protocol is applied on all the sources and sinks as default where
      <code>bootstrap.servers</code> config matches (for further details please see <code>spark.kafka.clusters.${cluster}.target.bootstrap.servers.regex</code>)

>       which can be overwritten with <code>kafka.security.protocol</code> config on a source or sink.

Reviewer: ```suggestion
      , and can be overridden by setting <code>kafka.security.protocol</code> on the source or sink.
```

>       , and can be overridden by setting <code>kafka.security.protocol</code> on the source or sink.

Reviewer: Well, to be nitpicky, the comma goes on the previous line.

Author: I've had similar feeling when a guy suggested this in a previous state, fixed :)
